### PR TITLE
FEATURE: RAIL-588 Filter examples of UI SDK

### PR DESCRIPTION
**Pull request by tomasreichmann**

### CODE REVIEW

**Comment by tomasreichmann** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/316/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1104/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/313/ : SUCCESS in 6m 02s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1103/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/882/ : SUCCESS in 8m 06s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/355/ : SUCCESS in 2m 46s

File: examples/package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "react-helmet": "5.2.0",
     "react-measure": "2.0.2",
     "react-router-dom": "4.2.2",
+    "react-select": "^1.2.1",
```
Comment (lhorak) [inline examples/package.json]: unnecessary ^

File: examples/package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "@gooddata/data-layer": "7.0.0-alpha0",
     "@gooddata/goodstrap": "55.0.0-alpha1",
     "@gooddata/react-components": "5.0.0-alpha3",
-    "gooddata": "6.0.0-alpha0",
+    "gooddata": "6.0.0-alpha3",
```
Comment (lhorak) [inline examples/package.json]: here is alpha-3 but in react-components is alpha-4

File: examples/src/utils/fixtures.js [hunk 3]
```diff
@@ -15,7 +15,18 @@ export const averageCheckSizeByServer = get(catalog, 'measures[Avg Check Size by
 export const monthDateIdentifier = get(catalog, 'dateDataSets[Date (Date)].attributes[Month (Date)].displayForms[Short (Jan) (Date)].identifier');
 export const dateDatasetIdentifier = get(catalog, 'dateDataSets[Date (Date)].identifier');
 
+export const locationStateAttributeIdentifier = get(catalog, 'attributes[Location State].identifier');
```
Comment (lhorak) [inline examples/src/utils/fixtures.js]: Look: https://github.com/gooddata/gooddata-react-components/blob/master/src/helpers/CatalogHelper.ts

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1111/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/890/ : SUCCESS in 2m 14s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/363/ : SUCCESS in 2m 24s

File: examples/package.json (hunk 1 above)
Comment (tomasreichmann) [reply examples/package.json]: VBO confirmed proper use of ^

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1116/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/895/ : SUCCESS in 1m 10s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/368/ : SUCCESS in 2m 21s

File: examples/package.json (hunk 1 above)
Comment (lhorak) [reply examples/package.json]: Sure, but the usage of striska is wanted in packages, that clients will add to their own projects, which is not the case of examples for react-components

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1120/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/899/ : SUCCESS in 3m 24s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/372/ : SUCCESS in 2m 11s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-merge-compliance-check/208/ : SUCCESS in 18s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/900/ : SUCCESS in 1m 08s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/322/ : SUCCESS in 5m 43s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/373/ : SUCCESS in 2m 51s

**Pull request merged**